app/android/app/build.gradle

```diff
@@ -47,7 +47,7 @@ android {
         applicationId "com.example.app"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion flutter.minSdkVersion
```

Comment:
Das ist sehr riskant, falls die nämlich in Zukunft hoch geht kann die App dann nicht mehr funktionieren

Reply:
Der Grund ist, dass ich min. 19 dafür brauche, die Version zu bekommen. Wenn dir was anderes einfällt, feel free

Reply:
Okay, dann werden wir das wohl mal so lassen, kann man ja irgendwann ändern falls nötig